checker.c

```diff
@@ -36,8 +37,33 @@ static BMS_parameters_s BMS_parameters_attributes = {
 		FALSE,
 		FALSE,
 		FALSE,
-		FALSE
+		FALSE,
+		LANGUAGE_ENGLISH
+};
+
+static print_Params_s print_Params_English = {
+	"Temperature",
+	"State of Charge",
+	"Charge Rate",
+	"out of range!",
+	"reaching Lower Threshold",
+	"reaching Higher Threshold"
 };
+
+static print_Params_s print_Params_German = {
+	"Temperieren",
+	"Stand der Abgabe",
+	"Gebührensatz",
+	"Außer Reichweite!",
+	"Erreichen der unteren Schwelle",
+	"Erreichen einer höheren Schwelle"
+};
+
+static print_Params_s printValues;
+static char printAllAttributes_Out_Of_Range[60] = " ";
+static char printAllAttributes_In_Lower_Threshold[60] = " ";
```

Comment:
Making this global means you will need to remember to reset it. A source of error during maintenance 
Any way to avoid global?